src/ngraph/ngraph_emitter.cc

```diff
@@ -1509,10 +1511,19 @@ void Emitter::CreateLossOps() {
     auto gradient = softmax - label;
 
     if (ignore) {
+      ngraph::Shape new_shape(gradient->get_shape().size(), 1);
```

Comment:
Could you please add a comment here about reshaping `mask` to allow broadcasting to the shape of gradient.